packages/venia-concept/_buildpack/create.js

```diff
@@ -133,7 +133,14 @@ function setDebugDependencies(fs, pkg) {
     );
     const overridden = {};
     const workspaceDir = resolve(__dirname, '../../');
-    fs.readdirSync(workspaceDir).forEach(packageDir => {
+    const extensionDir = resolve(workspaceDir, 'extensions');
+
+    const packageDirs = [
+        ...fs.readdirSync(workspaceDir),
+        ...fs.readdirSync(extensionDir).map(path => `extensions/${path}`)
+    ];
+
+    packageDirs.forEach(packageDir => {
         const packagePath = resolve(workspaceDir, packageDir);
         if (!fs.statSync(packagePath).isDirectory()) {
             return;
```

Comment:
If you use the Yarn method in my other comment, you don't need to do this resolve and stat check. `packageDirs` will contain only absolute-path directories.

Reply:
Much cleaner, implemented in 9a0ce84. This method additionally filters out workspaces without `package.json` files, so I was able to get rid of that check too.